### PR TITLE
Issue/43/phot error pipe

### PR DESCRIPTION
**Pull request by eacharles**
## Code Quality
- [ ] My code follows [the code style of this project](https://rail-hub.readthedocs.io/en/latest/source/contributing.html#naming-conventions)
- [ ] I have written unit tests or justified all instances of `#pragma: no cover`; in the case of a bugfix, a new test that breaks as a result of the bug has been added
- [ ] My code contains relevant comments and necessary documentation for future maintainers; the change is reflected in applicable demos/tutorials (with output cleared!) and added/updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] Any breaking changes, described above, are accompanied by backwards compatibility and deprecation warnings

### CODE REVIEW

File: src/rail/pipelines/utility/apply_phot_errors.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        Module='rail.creation.degraders.photometric_errors',
+    ),
+    roman = dict(
+        ErrorModel='RomanErrorModel',
+        Module='rail.creation.degraders.photometric_errors',
+    ),
+    #euclid = dict(
+    #    ErrorModel='EuclidErrorModel',
+    #    Module='rail.creation.degraders.photometric_errors',
+    #),
```
Comment (jfcrenshaw) [inline src/rail/pipelines/utility/apply_phot_errors.py:35]: Is Euclid supposed to be commented out?

File: src/rail/pipelines/utility/apply_phot_errors.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+)
+
+
+
+class ApplyPhotErrorsPipeline(RailPipeline):
```
Comment (jfcrenshaw) [inline src/rail/pipelines/utility/apply_phot_errors.py:40]: Do these not need a `run` method?

File: src/rail/pipelines/utility/apply_phot_errors.py (hunk 1 above)
Comment (eacharles) [reply src/rail/pipelines/utility/apply_phot_errors.py:35]: For now, yes..

File: src/rail/pipelines/utility/apply_phot_errors.py (hunk 2 above)
Comment (eacharles) [reply src/rail/pipelines/utility/apply_phot_errors.py:40]: I don't think so, I think that just come from the ceci base class

File: src/rail/pipelines/utility/apply_phot_errors.py (hunk 1 above)
Comment (eacharles) [reply src/rail/pipelines/utility/apply_phot_errors.py:35]: (We don't have data with roman and rubin and euclid all in the same file)

File: src/rail/pipelines/utility/apply_phot_errors.py (hunk 2 above)
Comment (jfcrenshaw) [reply src/rail/pipelines/utility/apply_phot_errors.py:40]: Ah I missed that these are pipelines and not stages. looks good

**Review by jfcrenshaw — APPROVED** [review]
LGTM

**Pull request merged**